alpha/lib/model/kuserPeer.php

```diff
@@ -571,25 +571,33 @@ public static function sendNewUserMail(kuser $user, $existingUser)
 	{
 		// setup parameters
 		$partnerId = $user->getPartnerId();
+		$partner = PartnerPeer::retrieveByPK($partnerId);
 		$userName = $user->getFullName();
-		if (!$userName) { $userName = $user->getPuserId(); }
+		if (!$userName)
+		{
+			$userName = $user->getPuserId();
+		}
 		$creatorUserName = 'Unknown';
 		if (!is_null(kCurrentContext::$ks_uid))
 		{
 			$creatorUser = kuserPeer::getKuserByPartnerAndUid($partnerId, kCurrentContext::$ks_uid);
-			if ($creatorUser) {
+			if ($creatorUser)
+			{
 				$creatorUserName = $creatorUser->getFullName();
 			}
 		}
-		$partner = PartnerPeer::retrieveByPK($partnerId);
+		
 		$publisherName = $partner->getName();
 		$loginEmail = $user->getEmail();
-		$roleName = $user->getUserRoleNames();
+		$roleNames = $user->getUserRoleNames();
 		$puserId = $user->getPuserId();
-		if (!$existingUser) {
-			$resetPasswordLink = UserLoginDataPeer::getPassResetLink($user->getLoginData()->getPasswordHashKey());
+		$useDynamicEmailTemplate = kEmails::getUseDynamicEmailTemplate($roleNames);
```

Comment:
change var name to match user role name meanning

Reply:
$roleNameToUseDynamicEmailTemplate